src/components/Content/RightPanel/TaskDetails/Wysiwyg/Wysiwyg.js

```diff
@@ -0,0 +1,103 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {EditorState, convertToRaw, convertFromRaw} from 'draft-js';
+import Editor from 'draft-js-plugins-editor';
+import createAutoListPlugin from 'draft-js-autolist-plugin'
+import createMarkdownPlugin from 'draft-js-markdown-plugin';
+import {markdownToDraft, draftToMarkdown} from 'markdown-draft-js';
+import ReactMarkdown from 'react-markdown';
+import './Wysiwyg.css';
+
+const autoListPlugin = createAutoListPlugin();
+
+const plugins = [
+  autoListPlugin,
+  createMarkdownPlugin()
+];
+
+class Wysiwyg extends Component {
+  componentWillMount() {
+    const {tasks: { taskMap }, taskGid, attributeName} = this.props,
+      attributeValue = taskMap[taskGid][attributeName],
```

Comment:
@4xdk Got a more description variable name than `attributeValue`?

Reply:
Well, this variable can hold any attribute value - depending on props passed to Wysiwyg element. I think generic variable name fits in this case.